### PR TITLE
Minor change in README.md

### PR DESCRIPTION
**Pull request by liulanz**
I changed the link for "JOIN THE COMMUNITY" to [spectrum chat channel](https://spectrum.chat/next-js?tab=posts). 

It was orginally directed to the [Discussions](https://github.com/zeit/next.js/discussions), but the label says "join us in spectrum".

I noticed that "alpha" branch does has the correct link to the spectrum chat. I think it would be better for the master branch to also have correct link.

### CODE REVIEW

File: packages/next/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <a aria-label="License" href="https://github.com/zeit/next.js/blob/canary/license.md">
     <img alt="" src="https://img.shields.io/npm/l/next.svg?style=for-the-badge&labelColor=000000">
   </a>
-  <a aria-label="join us in spectrum" href="https://github.com/zeit/next.js/discussions">
+  <a aria-label="join us in spectrum" href="https://spectrum.chat/next-js?tab=posts">
```
Comment (lfades) [inline packages/next/README.md]:
```suggestion
  <a aria-label="Join the community on GitHub" href="https://github.com/zeit/next.js/discussions">
```

**Comment by lfades** [comment]
@liulanz Please commit my suggestion and I'll approve the PR, the community is in GitHub Discussions, not in Spectrum. I've updated the aria label to mention GitHub instead.

**Comment by liulanz** [comment]
Sorry about the link, and thank you for the help!

**Review by lfades — APPROVED** [review]
Thank you :100:

**Review by lfades — CHANGES_REQUESTED** [review]
Oh something I didn't notice, you changed the branch from `canary` to `master`, can you change it to `canary` again :pray:

**Review by lfades — APPROVED** [review]

**Pull request merged**